layouts/partials/site-header.html

```diff
@@ -66,7 +66,12 @@ <h3 class="font-bold text-3xl lg:text-5xl">{{ .Title }}</h3>
                 </div>
                 {{ if .Params.subheadertext }}
                 {{ with .Params.subheadertext }}
-                <p class="text-2xl lg:text-4xl mt-1 lg:mt-2 font-extralight">{{ . }}</p>
+                <p class="hidden sm:block text-2xl lg:text-4xl mt-1 lg:mt-2 font-extralight">{{ . }}</p>
```

Comment:
I thought it would be best to hide subheadertext on small screens because sometimes it becomes too difficult to read.